### PR TITLE
Add /v0.1/apps endpoint

### PR DESCRIPTION
**Pull request by tomorrowkey**
Add a method to access `/v0.1/apps` endpoint.

```ruby
client = Bitrise::Client.new(access_token: ENV["BITRISE_ACCESS_TOKEN"])
client.apps.paging
=> #<Bitrise::Paging:0x000000010590a500 @page_item_limit=50, @total_item_count=7>
client.apps.data.size
=> 7
```

### CODE REVIEW

**Comment by tomorrowkey** [comment]
@mataku Please review this 😉

File: lib/bitrise/response.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+module Bitrise
+  class Response
+    def initialize(json)
+      @json = json
+    end
+  end
+
+  class Paging
+    attr_reader :total_item_count, :page_item_limit
```
Comment (mataku) [inline lib/bitrise/response.rb]: `next` is also necessary?

If there are more apps than the number specified by `limit` query parameter in request, a request with second page onwards is necessary.  In that case, it should make a paging request with the slug returned by the Bitrise API as the `paging` request parameter.

https://api-docs.bitrise.io/#/application/app-list

Comment (tomorrowkey) [reply lib/bitrise/response.rb]: Yes, I missing it. I will fix it 🔥

Comment (tomorrowkey) [reply lib/bitrise/response.rb]: `next` is added at [83d0a8f](https://github.com/mataku/bitrise-client/pull/6/commits/83d0a8ff4524e2774f25dd14649242037b1b50da)

**Review by mataku — APPROVED** [review]
🙆

**Comment by mataku** [comment]
Released as v0.3.1. thanks!

**Pull request merged**